modules/admin_manual/pages/installation/docker/index.adoc

```diff
@@ -96,143 +111,178 @@ ownclouddockerserver_owncloud_1   … /usr/bin/owncloud server  Up      0.0.0.0:
 ownclouddockerserver_redis_1      … /bin/s6-svscan /etc/s6    Up      {std-port-redis}/tcp
 ----
 
-In it, you can see that the database, ownCloud, and Redis containers are running, and that only ownCloud is accessible via port {std-port-http} on the host machine.
+In it, you can see that the database, ownCloud, and Redis containers are running, and that only
```

Comment:
Only? Sounds weird, not needed.

Reply:
No worry, this is just a editoring linebreak for better readbility. Else the line would get long and you have to shift the view via the slider to read the complete text in one shot, which is quite hard if you have long texts. When rendering, everything gets put together correctly.
In Antora you could write:
H
e
l
l
o
and it would render to Hello, except you force a hard linebreak with eg `+`

Reply:
Misunderstood :)

> and that only ownCloud is accessible via port 

I think we don't need the word only here

Reply:
fixed and squashed